config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-trusted.yaml

```diff
@@ -22,6 +22,29 @@ periodics:
       args:
       - -c
       - "cd groups && make run -- --confirm"
+- name: ci-k8sio-groups-head
+  interval: 1h
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  max_concurrency: 1
+  annotations:
+    testgrid-dashboards: wg-k8s-infra-k8sio, sig-testing-canaries
+    testgrid-alert-email: k8s-infra-alerts@kubernetes.io
+    testgrid-num-failures-to-alert: '1'
+  extra_refs:
+  - org: kubernetes
+    repo: k8s.io
+    base_ref: HEAD
+  spec:
+    serviceAccountName: gsuite-groups-manager
+    containers:
+    - name: groups
+      image: golang:1.13
+      command:
+      - bash
+      args:
+      - -c
+      - "cd groups && make run -- --confirm"
```

Comment:
one thing that isn't clear just looking at the PR is the other intent of the job vs trying HEAD / why we need a new job here.

this looks like it is a copy+paste of a "production" job just with HEAD?
are there no issues running both?

Reply:
there are not, and yes that was the intent, I wanted a repo that had its default branch renamed